src/Controller/Admin/Improve/Design/LinkBlockController.php

```diff
@@ -260,6 +260,13 @@ private function processForm(Request $request, $successMessage, $linkBlockId = n
             }
 
             $this->flashErrors($saveErrors);
+        } else {
+            $formErrors = [];
+            foreach ($form->getErrors(true) as $error) {          
+                $formErrors[] = $error->getMessage();             
+            }
+            
+            $this->flashErrors($formErrors);
```

Comment:
```suggestion
        if ($form->isSubmitted()) {
            if ($form->isValid()) {
                $data = $form->getData();
                $saveErrors = $formHandler->save($data);
                if (0 === count($saveErrors)) {
                    $this->addFlash('success', $this->trans($successMessage, 'Admin.Notifications.Success'));
                    return $this->redirectToRoute('admin_link_block_list');
                }
            }

            $formErrors = [];
            foreach ($form->getErrors(true) as $error) {
                $formErrors[] = $error->getMessage();
            }

            $this->flashErrors($formErrors);
        }
```

Reply:
The whole code should be something like this

Reply:
Thank you 💪